components/resources/wallet_strings.grdp

```diff
@@ -132,6 +132,17 @@
   <message name="IDS_BRAVE_WALLET_OTHERS" desc="Swap max slippage others label">Others: <ph name="SLIPPAGE_PERCENT"><ex>1</ex>$1</ph>%</message>
   <message name="IDS_BRAVE_WALLET_SUGGESTED" desc="Swap max slippage suggested option">Suggested</message>
   <message name="IDS_BRAVE_WALLET_CUSTOM" desc="Swap max slippage custom option">Custom</message>
+  <message name="IDS_BRAVE_WALLET_CHEAPEST" desc="Swap route option cheapest label">Cheapest</message>
+  <message name="IDS_BRAVE_WALLET_FASTEST" desc="Swap route option fastest label">Fastest</message>
+  <message name="IDS_BRAVE_WALLET_ON_NETWORK" desc="Swap token on a specific network">on <ph name="NETWORK_NAME"><ex>Ethereum Mainnet</ex>$1</ph></message>
+  <message name="IDS_BRAVE_WALLET_ROUTES" desc="Swap routes title">Routes</message>
+  <message name="IDS_BRAVE_WALLET_SWAP_ON_NETWORK_VIA_EXCHANGE" desc="Swap on network via an exchange">Swap on <ph name="NETWORK_NAME"><ex>Ethereum Mainnet</ex>$1</ph> via <ph name="EXCHANGE_NAME"><ex>0x</ex>$2</ph></message>
+  <message name="IDS_BRAVE_WALLET_BRIDGE_FROM_NETWORK_TO_NETWORK" desc="Bridge from network name to network name">Bridge from <ph name="FRIST_NETWORK_NAME"><ex>Ethereum Mainnet</ex>$1</ph> to <ph name="SECOND_NETWORK_NAME"><ex>Polygon Mainnet</ex>$2</ph></message>
+  <message name="IDS_BRAVE_WALLET_EXCHANGE_NAME_PLUS_STEPS" desc="Name of exchange plust steps to complete a swap"><ph name="EXCHANGE_NAME"><ex>0X</ex>$1</ph> + <ph name="EXCHANGE_STEPS"><ex>8</ex>$2</ph> <ph name="STEPS_OR_STEP"><ex>steps</ex>$3</ph></message>
+  <message name="IDS_BRAVE_WALLET_STEP" desc="Single step in a swap">step</message>
+  <message name="IDS_BRAVE_WALLET_STEPS" desc="Multiple steps in a swap">steps</message>
```

Comment:
Could we use a ICU plural string here instead? Here's an example: https://github.com/brave/brave-core/blob/master/components/resources/rewards_strings.grdp#L92-L95

Reply:
@mkarolin I have added a `PlurStringHandler` to the `WalletUI` code base and implemented these changes here https://github.com/brave/brave-core/compare/0c81790753299477422575139eb4965d5d3ac684..c829557088de751f1023ddf83858cd0cf7671688